.github/release-drafter.yml

```
@@ -11,8 +11,19 @@ categories:
   - title: 🐛 Bug Fixes
     labels:
       - bug
+  - title: Robutness
+    labels:
+      - 'robustness'
+  - title: Library Integration
```

```suggestion
  - title: ⤵️ Library Integration
```

Or another from https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md#arrow